src/Interface/ShapeHelperOpInterface.td

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//===- ShapeHelperInterface.td - ShapeHelper Interface -*- tablegen -===//
```

Comment:
Please expand it up to 80 characters.

Reply:
Sure, since there is already a follow up PR where I add the interface to all but the ops that accept Seq and Opt, let me make the change there.